posthog/admin/__init__.py

```diff
@@ -1,7 +1,25 @@
 from django.contrib import admin
 
 from posthog.admin.admins import *
```

Comment:
I'd not do star imports in general 🙈

Reply:
Got it. Removed here and want to add a linter removing them in general (we hardly use star imports): https://github.com/PostHog/posthog/pull/20915